test/nuts/local/localTrackingFileMovesImage.nut.ts

```diff
@@ -75,5 +79,7 @@ describe('it detects image file moves ', () => {
     expect(await repo.getChangedFilenames())
       .to.be.an('array')
       .with.lengthOf(0);
+
+    delete process.env.SF_BETA_TRACK_FILE_MOVES;
```

Comment:
nit: I have a slight preference for doing these in an `afterEach` so that they next test added doesn't have to remember to unset it.